api/src/exceptions/NotImplementedException.ts

```diff
@@ -0,0 +1,17 @@
+import HttpException from './HttpException';
+
+/**
+ * Not implemented exception.
+ */
+class NotImplementedException extends HttpException {
+    /**
+     * Create a NotImplementedException.
+     * @param routeName Route that is not implemented.
+     */
+    constructor(routeName: string) {
+        super(404, `Route ${routeName} is not implemented`);
+        Object.setPrototypeOf(this, NotImplementedException.prototype);
```

Comment:
Is this an attempt so that we can check if an object is of type `NotImplementedException` (instead of its parent class)?

Reply:
That is correct. I added some unit tests to `exception.test.ts` that demonstrate what it allows us to do.